AK/Unaligned.h

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 2018-2021, Andreas Kling <kling@serenityos.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/Concepts.h>
+#include <AK/Types.h>
+
+template<Integral T>
+void read_possibly_unaligned_data(u8* where, T& data) requires(sizeof(T) == 8 || sizeof(T) == 4 || sizeof(T) == 2)
```

Comment:
The read part of this is already in AK in ByteReader.h. could we get some reuse going?

Reply:
Ah then I'll just use that then